client/my-sites/themes/thanks-modal.scss

```diff
@@ -48,11 +48,17 @@
 		+ span {
 			display: block;
 			text-align: center;
-			color: var( --color-neutral-20 );
+			color: var( --color-text-subtle );
```

Comment:
This is separate and unrelated, but `--color-neutral-20` fails contrast requirements. Most of these were updated a while ago to `--color-text-subtle`, but I suspect this one was just missed.

<img width="137" alt="Screenshot 2020-07-14 at 11 18 33" src="https://user-images.githubusercontent.com/43215253/87414509-cbfef600-c5c3-11ea-9224-125c2f716212.png">